### PR TITLE
rebuild with latest starport, move in the Makefile & readme

### PR DESCRIPTION
**Pull request by faddat**

### CODE REVIEW

**Comment by faddat** [comment]
Closes #7

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -29,11 +29,11 @@ jobs:
       - name: Compile
         run: |
           go mod download
-          cd cmd/junod
+          cd cmd/Junod
```
Comment (jackzampolin) [inline .github/workflows/build.yml:32]: why `J` instead of `j`

Comment (faddat) [reply .github/workflows/build.yml:32]: consistency:

Repo name is:

github.com/CosmosContracts/Juno

Comment (faddat) [reply .github/workflows/build.yml:32]: but heh, seems everywhere else, it's lowercase.  

:P

I'll lowercase this one for consistency

:P

**Comment by faddat** [comment]
Here's a proposal for this one, why don't we wait until the final release of Starport 16 and then do this. Let me know what you think about the capitalization because obviously it's quite easy to do either way.

**Comment by JakeHartnell** [comment]
> Here's a proposal for this one, why don't we wait until the final release of Starport 16 and then do this. Let me know what you think about the capitalization because obviously it's quite easy to do either way.

Seems like it's out. : )

Lower case is better please. :pray:

Thanks for this btw! :tada:

**Comment by faddat** [comment]
All set

**Comment by faddat** [comment]
You know last night I probably forgot the address prefix

File: app/prefix.go [hunk 2]
```diff
@@ -5,7 +5,7 @@ import (
 )
 
 const (
-	AccountAddressPrefix = "juno"
+	AccountAddressPrefix = "cosmos"
```
Comment (JakeHartnell) [inline app/prefix.go:8]: Shouldn't this be `juno`?

Comment (faddat) [reply app/prefix.go:8]: Yes

**Review by JakeHartnell — APPROVED** [review]
LGTM

**Pull request merged**